versions/3.0.4.md

```diff
@@ -1373,7 +1373,7 @@ Field Name | Type | Description
 <a name="mediaTypeSchema"></a>schema | [Schema Object](#schemaObject) \| [Reference Object](#referenceObject) | The schema defining the content of the request, response, or parameter.
 <a name="mediaTypeExample"></a>example | Any | Example of the media type.  The example object SHOULD be in the correct format as specified by the media type.  The `example` field is mutually exclusive of the `examples` field.  Furthermore, if referencing a `schema` which contains an example, the `example` value SHALL _override_ the example provided by the schema.
 <a name="mediaTypeExamples"></a>examples | Map[ `string`, [Example Object](#exampleObject) \| [Reference Object](#referenceObject)] | Examples of the media type.  Each example object SHOULD  match the media type and specified schema if present.  The `examples` field is mutually exclusive of the `example` field.  Furthermore, if referencing a `schema` which contains an example, the `examples` value SHALL _override_ the example provided by the schema.
-<a name="mediaTypeEncoding"></a>encoding | Map[`string`, [Encoding Object](#encodingObject)] | A map between a property name and its encoding information. The key, being the property name, MUST exist in the schema as a property. The encoding object SHALL only apply to `requestBody` objects when the media type is `multipart` or `application/x-www-form-urlencoded`.
+<a name="mediaTypeEncoding"></a>encoding | Map[`string`, [Encoding Object](#encodingObject)] | A map between a property name and its encoding information. The key, being the property name, MUST exist in the schema as a property. The encoding object SHALL only apply to `requestBody` objects when the media type is `multipart` or `application/x-www-form-urlencoded`.  If no Encoding Object is provided for a relevant parameter, the behavior is determined by the default values documented for the Encoding Object.
```

Comment:
I have some questions/concerns:

The statement
> The encoding object SHALL only apply to `requestBody` objects ...

is confusing to me, because the "encoding object" is the value part of the map entries of "encoding", which applies to properties of a schema.  Later in the spec I see this statement:
> An encoding attribute ... is only applicable to multipart and application/x-www-form-urlencoded request bodies.

Should "encoding object" actually be "encoding attribute" (or "encoding field") in the originally cited text?

Also confusing for me is the word "parameter" in the sentence:
> If no Encoding Object is provided for a relevant parameter ...

From the earlier text, I thought that encoding objects were provided for _properties_ of a schema -- not parameters. Maybe this is nit picking words but in OAS these are very different things. Secondarily, I'm not sure what "relevant" means in this context. When would a property be "irrelevant"? And if there are "irrelevant" properties, then shouldn't this qualification be added above:
> The key, being the _relevant_ property name,

I think the following suggested text would resolve my concerns:
```suggestion
<a name="mediaTypeEncoding"></a>encoding | Map[`string`, [Encoding Object](#encodingObject)] | A map between a property name and its encoding information. The key, being the property name, MUST exist in the schema as a property. The encoding attribute SHALL only apply to `requestBody` objects when the media type is `multipart` or `application/x-www-form-urlencoded`.  If no Encoding Object is provided for a property, the behavior is determined by the default values documented for the Encoding Object.
```